data/boarding missions.txt

```diff
@@ -24,6 +24,343 @@ mission "Assisting Merchant"
+				decline
+	destination Earth	
+	
+mission "Assisting Pirates (Thief)"
+	assisting
+	repeat
+	to offer
+		random < 12
+	source
+		government Pirate
+	on offer
+		payment -100000
+		conversation
+			`After assisting the <origin> you find that 100000 credits have been removed from your account by those blasted pirates.`
+				decline
+	destination Greenrock
+
+mission "Assisting Pirates (Setback)"
+	assisting
+	repeat
+	to offer
+		random < 8
+	source
+		government Pirate
+	on offer
+		payment 100
+		conversation
+			`The captain of the <origin> gives you <payment> for "helping him out of a temporary setback."`
+				decline
+	destination Greenrock
+	
+mission "Assisting Pirates (Threat)"
+	assisting
+	repeat
+	to offer
+		random < 8
+	source
+		government Pirate
+	on offer
+		payment 100
+		conversation
+			`The captain of the <origin> gives you <payment> and a message saying, "You better watch who you mention this to, I'll be watching."`
+				decline
+	destination Greenrock
+
+mission "Assisting Pirates (Weed)"
+	assisting
+	repeat
+	to offer
+		random < 8
+	source
+		government Pirate
+	on offer
+		payment 100
+		conversation
+			`The captain of the <origin> gives you <payment> along with lucrative offer for space weed. "Only 6000 Credits a Pound!"`
+				decline
+	destination Greenrock
+	
+mission "Assisting Syndicate"
+	assisting
+	repeat
+	to offer
+		random < 15
+	source
+		government Syndicate
+	on offer
+		payment 5000
+		conversation
+			`Knowing you risked your life to save the <origin> and help the Syndicate cause, the captain gives you <payment>.`
+				decline
+	destination Foundry
+
+mission "Assisting Syndicate (Protector)"
+	assisting
+	repeat
+	to offer
+		random < 15
+	source
+		government Syndicate
+	on offer
+		payment 5000
+		conversation
+			`Knowing you risked your life to save the <origin> and help the Syndicate cause, the captain gives you <payment> along with a message reading, "Trouble with Pirates? Hire a Syndicate Protector and your troubles will be over."`
+				decline
+	destination Foundry
+
+mission "Assisting Syndicate (Catalog)"
+	assisting
+	repeat
+	to offer
+		random < 15
+	source
+		government Syndicate
+	on offer
+		payment 5000
+		conversation
+			`Knowing you risked your life to save the <origin> and help the Syndicate cause, the captain gives you <payment> and the latest copy of the Syndicated Systems Catalog.`
+				decline
+	destination Foundry
+	
+mission "Assisting Syndicate (Dental)"
+	assisting
+	repeat
+	to offer
+		random < 15
+	source
+		government Syndicate
+	on offer
+		payment 5000
+		conversation
+			`Knowing you risked your life to save the <origin> and help the Syndicate cause, the captain gives you <payment> along with a message reading, "Work for the Syndicate! Our dental plans will keep you smiling!"`
+				decline
+	destination Foundry
+	
+mission "Assisting Oathkeepers"
```

Comment:
You could just combine the Oathkeeper and Navy missions by having the source be `government Republic "Nacy (Oathkeeper)"`. The Oathkeepers and the rest of the Navy have the same job (the former group just having specific location in which they do it), so the conversations could be the same.